### PR TITLE
Execute-phase-mvp

### PR DESCRIPTION
**Pull request by ricglz**

### CODE REVIEW

File: src/main/java/codeu/helper/BotActions.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+public class BotActions {
+
+    private ActivityStore activityStore;
+    private User user;
+    private ConversationStore conversationStore;
+    private HttpServletResponse response;
+    private MessageStore messageStore;
+    private UserStore userStore;
+
+    public BotActions(String username) {
```
Comment (ksingarajah) [inline src/main/java/codeu/helper/BotActions.java]: It might be easier to just use the store's respective getInstance() methods.

Comment (ricglz) [reply src/main/java/codeu/helper/BotActions.java]: Right. Thought I did it, thanks.

**Review by tsalako — CHANGES_REQUESTED** [review]
I know we talked over chat but I'll reiterate on here. One thing we are missing is adding the message to the message store for this conversation. So along with executing we do need to generate a string to add to the conversation.

**Review by manjilgtm — APPROVED** [review]
All the botActions code are clear and concise. LGTM. We could definitely add more actions later.

**Pull request merged**